From c8273c251b029f1c2c127bd3fa9d6d83359a60d8 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Sun, 18 May 2025 09:29:05 +0300
Subject: [PATCH] [SW-227076] g2e: update hbm rl config values

Experiments done to fine-tune HBM RL configuration values
to limit HBM BW to 1.4 TB/s, led to the following values:
max_sat = 5, reset_val = 52 & max_timeout = 99.

Change-Id: I02ff509a772fdbb38b084d37e6013bad63db2a79
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/arch/gaudi2/device.c b/arch/gaudi2/device.c
index bc103a1..5cba941 100644
--- a/arch/gaudi2/device.c
+++ b/arch/gaudi2/device.c
@@ -1332,9 +1332,9 @@
 #define GAUDI2E_HBM_RL_TIMEOUT_MASK		0xFF0000	/* Bits 16..23 */
 #define GAUDI2E_HBM_RL_EN_SHIFT			24		/* RL enable */
 #define GAUDI2E_HBM_RL_EN_MASK			0x1000000	/* Bit 24 */
-#define GAUDI2E_HBM_RL_MAX_SAT			5
-#define GAUDI2E_HBM_RL_RESET_VAL		11
-#define GAUDI2E_HBM_RL_MAX_TIMEOUT		19
+#define GAUDI2E_HBM_RL_MAX_SAT			7
+#define GAUDI2E_HBM_RL_RESET_VAL		52
+#define GAUDI2E_HBM_RL_MAX_TIMEOUT		99
 #define GAUDI2E_HBM_RL_DCORE_NUM		4
 #define GAUDI2E_HBM_RL_RTR_NUM			8
 #define DCORE_N_OFFSET				0x200000
